src/components/dashboard/DepositTokens.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import IconSection from './IconSection'
+import ColoredSection from './ColoredSection'
+
+import './DepositTokens.scss'
+
+class DepositTokens extends React.Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
+    return (
+      <ColoredSection styleName="root"
+        head={this.renderHead()}
+        headStyle={this.props.headStyle}
+      />
+    )
+  }
+
+  renderHead() {
+    return (
+      <div>
+        <IconSection title={this.props.title}>
+          <div styleName="balance">
+            <div styleName="label">Your time deposit:</div>
+            <div styleName="value">
+              <span styleName="value1">1 512 000</span>
```

Comment:
must be independent component for currency with input props like `value='100000.0000'`